MIP104/MIP104.md

```diff
@@ -540,161 +347,59 @@ Current target exposure of Morpho sUSDe or USDe vault is: 0
 
-MakerDAO must maintain a reserve of at least 20% of the total Dai Supply held as Cash Stablecoins, or assets and exposure that can rapidly be converted to Cash Stablecoins. Whenever the total amount of USDC in the PSM, plus the total amount of USDC in GUNI pools falls below 400 million, USDC must be transferred from Coinbase Custody to the PSM to bring the total amount of USDC in the PSM, plus the total amount of USDC in the GUNI pools, to 550m USDC. Whenever the total amount of USDC in the PSM, plus the total amount of USDC in GUNI pools exceeds 700 million, USDC must be transferred from the PSM to Coinbase Custody to bring the total amount of USDC in the PSM, plus the total amount of USDC in the GUNI pools, to 550m USDC. This mechanism can be fully replaced with an upgraded PSM that obsoletes the Coinbase Custody solution. The Stability Facilitator can directly trigger an executive vote to implement an upgraded PSM.
+#### 7.2: Asset Liability Management
 
-##### 7.2.1.3: Cash Stablecoin Definition
+To ensure adequate liquidity, Cash Stablecoins are managed through a simple ALM framework that allocates excess Cash Stablecoins into low-risk RWA assets, and draw on these assets to replenish the Cash Stablecoins when they fall too low.
 
-A Cash Stablecoin is a stablecoin listed in *7.2.1.3.1A* or a technically secure DeFi token that can be instantly converted to a listed stablecoin, or a custody solution that can be converted into a listed stablecoin within 30 minutes. The Stability Facilitators can modify *7.2.1.3.1A* through the weekly cycle to react to changing market conditions, opportunities or risks. This can include adding or removing Cash Stablecoins, or changing the terms of existing Cash Stablecoins on the list.
+7.2.1: Excess Cash Stablecoins
 
-###### 7.2.1.3.1A
+If the Dai Collateral portfolio contains more than 30% of Cash Stablecoins as defined above, then any Cash Stablecoins above the 30% mark must be allocated towards the collateral defined in *7.2.3*
 
-¤¤¤
+##### 7.2.2: Lack of Cash Stablecoins
 
-List of Cash Stablecoins:
+If the Dai Collateral portfolio contains less than 20% Cash Stablecoins, the assets defined in *7.2.3* must be sold off to return the Cash Stablecoin exposure to 25%.
 
-- USDC and USDC in Coinbase Custody
+##### 7.2.3: Low Risk RWA
 
-¤¤¤
+Clydesdale and andromeda are two RWA Arranged Structures that can allocate capital into Low-Risk RWA (LRR), safe, short term treasury strategies of less than 1 year duration. Whenever excess Cash Stablecoins must be allocated into LRR, or whenever LRR must be sold to replenish Cash Stablecoins, the amounts allocated or sold are split equally among the two RWA Arranged Structures. As as a result, the two RWA Arranged Structures should generally always track the same amount of exposure into LRR, although short term discrepancies for operational reasons are acceptable.
```

Comment:
"As a result" repeats the word "as".

Reply:
Capital letter "A" for Andromeda at the beginning of the section.